viewmaster/viewmaster.py

```
@@ -2065,9 +2373,19 @@ def build_local_cache():
                      ip_address)
         return 'Viewmaster cache building FAILED'
 
-def fill_page_cache():
-    """Load all the top-level and large pages into the cache. This could take
-    a while."""
+def fill_page_cache(logger):
```

Ruff (0.14.11)

Unused function argument: `logger`

(ARG001)